components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -756,7 +756,14 @@ class PageWallet extends React.Component<Props, State> {
 
     // ledger::type::Result::CORRUPTED_DATA
     if (walletRecoveryStatus === 17) {
-      return <span dangerouslySetInnerHTML={{ __html: getLocale('walletRecoveryOutdated') }} />
+      const tags = getLocaleTags('walletRecoveryOutdated')
+      return <span>
+        {tags.beforeTag}
+        <a href='https://brave.com/faq#convert-old-keys' target='_blank' rel='noopener noreferrer'>
```

Comment:
to test: go to brave://rewards, click settings, restore key, enter 'a a a a a a a a a a a a a a a a' for the key. make sure that it shows a message that the key is no longer supported and that clicking on the link in the message opens a new tab.